### DISCUSSION TITLE
How to save a row from Multiselect?

### DISCUSSION THREAD

**Discussion by Sagestum** [Budibase/budibase#11627] [Help]
I have written a form for new notes on a certain entry in the database. For this, I would like to select the designated employee in a multiselect picker. 

So far so good, but when I now want to save this via a "save Row" button, I get an error message. 

"Employee Field not in list"

I don't quite understand what the tool is trying to tell me. 

I can select the employees directly in the database. I think I have to make a text list out of the array somehow, at least that's what I assume, but what's the smartest way to do that?

![Bildschirmfoto 2023-08-30 um 20 40 56](https://github.com/Budibase/budibase/assets/11883823/dc6f6ffc-2748-45da-8224-91f6042d8f01)
![Bildschirmfoto 2023-08-30 um 20 42 05](https://github.com/Budibase/budibase/assets/11883823/40534d8e-b754-407f-a27d-682df6909669)
![Bildschirmfoto 2023-08-30 um 20 44 15](https://github.com/Budibase/budibase/assets/11883823/ad19d4c6-af52-4b9c-b925-cdd249ad59f4)

Thanks for your help 

Chris

**Comment by melohagan** [comment]
Hey @Sagestum 

That error occurs when the value is not found in the list of options defined in the multi-select field.

![Screenshot 2023-09-05 at 11 55 51](https://github.com/Budibase/budibase/assets/101575380/141641f4-a950-49a1-a19b-8e4abf1bfecd)

![Screenshot 2023-09-05 at 12 01 34](https://github.com/Budibase/budibase/assets/101575380/0e9a20d2-ec8f-4651-840f-bf64ce4a3ab8)

![Screenshot 2023-09-05 at 12 01 55](https://github.com/Budibase/budibase/assets/101575380/48e281a5-7038-4a13-b673-b5cb4dc87bc6)

Therefore, I suspect that whatever value `{{ NeueNotiz.Fields.Mitarbeiter }}` is resolving to is not in the defined multi-select options list.

What field type is `NeueNotiz.Fields.Mitarbeiter` ? Are you sure the value would match?
